### PR TITLE
Fix watch task

### PR DESCRIPTION
**Pull request by locriandev**

### CODE REVIEW

File: artbotlib/help.py [hunk 1]
```diff
@@ -21,7 +21,9 @@ def show_help(so):
 * What kernel is used in `release image name or pullspec`?
 
 _*ART release/Brew monitoring:*_
-* Watch `release_url|brew_url|prow_job_url`
```
Comment (thegreyd) [inline artbotlib/help.py:24]: I would like to keep the "Watch <url>" interface - it's intuitive and easy to remember
We can remove the ID regex

**Review by thegreyd — CHANGES_REQUESTED** [review]
Would like to keep "watch [url]" for brew.

File: artbotlib/help.py (hunk 1 above)
Comment (locriandev) [reply artbotlib/help.py]: done

**Pull request merged**